src/setupTests.tsx

```diff
@@ -2,3 +2,22 @@
 
 import 'jest-styled-components'
 import 'react-testing-library/cleanup-after-each'
+
+// Supress "test was not wrapped in act()" Warning
+// Remove when upgrading to react-dom@16.9.0
+// Bug: https://github.com/facebook/react/pull/14853
+// Fix: https://github.com/testing-library/react-testing-library/issues/281#issuecomment-480349256
+/* tslint:disable:no-console no-any */
+const originalError = console.error
+beforeAll(() => {
+  console.error = (...args: any[]) => {
+    if (/Warning.*not wrapped in act/.test(args[0])) {
```

Comment:
wow, that's some heavy duty suppression! Are we sure we care about suppressing warnings, given risk of overriding `console.error`?

Reply:
This is only in testing, and only filters this one error. 

Alternately, we can remove this and just ignore the errors as I believe they are non-blocking.